include/rnp.h

```diff
@@ -65,6 +80,11 @@ typedef struct rnp_t {
 int rnp_init(rnp_t *);
 int rnp_end(rnp_t *);
 
+/* init, reset and free rnp operation context */
+int  rnp_ctx_init(rnp_ctx_t *);
+void rnp_ctx_reset(rnp_ctx_t *);
+void rnp_ctx_free(rnp_ctx_t *);
+
 /* debugging, reflection and information */
 int         rnp_set_debug(const char *);
```

Comment:
Ah, I see. I now struggling with clang-format - version 5.0 which is automatically installed on Mac by brew produces incompatible with 4.0 output, so travis tests fail.